qemu/tests/sgx_cpu.py

```diff
@@ -0,0 +1,49 @@
+from virttest import error_context
+from virttest.utils_misc import verify_dmesg
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu sgx cpu test:
+    1. Boot sgx VM
+    2. Check sgx guest cpuid with sub-features
+    3. Check sgx guest corresponding CPUID entries
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    error_context.context("Start sgx cpu test", test.log.info)
+    timeout = params.get_numeric("login_timeout", 240)
+
```

Comment:
Do we want to do a sgx host caps check before starting a sgx VM? Before I thought it's needed, but now I dont think it's a must, what's your opinion?

Reply:
I remove host caps check since it didn't require host numa caps, I think it is ok to add host epc caps check here. But note that some sgx cases will use profile to pass sgx params with existing cases, e.g. passthrough, nested. Those cases won't have host epc cap check.

Reply:
Then it's OK for me to remove the check